hi-ml-azure/src/health/azure/azure_util.py

```diff
@@ -793,3 +793,64 @@ def is_local_rank_zero() -> bool:
     global_rank = os.getenv(ENV_GLOBAL_RANK)
     local_rank = os.getenv(ENV_LOCAL_RANK)
     return global_rank is None and local_rank is None
+
+
+def download_from_datastore(datastore_name: str, prefix: str, output_path: Path,
+                            aml_workspace: Optional[Workspace] = None,
+                            workspace_config_path: Optional[Path] = None,
+                            overwrite: bool = False,
+                            show_progress: bool = False) -> None:
+    """
+    Download a file from an Azure ML Datastore that is registered within a given Workspace.
+
+    If not running inside AML and neither a workspace nor the config file are provided, the code will try to locate a
+    config.json file in any of the parent folders of the current working directory. If that succeeds, that config.json
+    file will be used to create the workspace.
+
+    :param datastore_name: The name of the Datastore containing the blob to be downloaded. This Datastore itself
+        must be an instance of an AzureBlobDatastore.
+    :param prefix: The prefix to the blob to be downloaded
+    :param output_path: The path to which the blob should be downloaded
+    :param aml_workspace: Optional Azure ML Workspace object
+    :param workspace_config_path: Optional path to settings for Azure ML Workspace
+    :param overwrite: If True, will overwrite any existing file at the same remote path
```

Comment:
for `overwrite`, can you also say what the "False" behaviour is? I find it important to document both True and False of _any_ boolean parameters, because we can't expect people to know the default behaviour otherwise.